docs/guide/ingress/annotations.md

```diff
@@ -790,16 +790,19 @@ TLS support can be controlled with the following annotations:
 
 - <a name="mutual-authentication">`alb.ingress.kubernetes.io/mutual-authentication`</a>  specifies the mutual authentication configuration that should be assigned to the Application Load Balancer secure listener ports. See [Mutual authentication with TLS](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/mutual-authentication.html) in the AWS documentation for more details.
 
-    !!!note "Configuration Options"
-        - `port: listen port ` 
-            - Must be a HTTPS port specified by [listen-ports](#listen-ports).
-        - `mode: "off" (default) | "passthrough" | "verify"`
-            - `verify` mode requires an existing trust store resource.
-            -  See [Create a trust store](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/mutual-authentication.html#create-trust-store) in the AWS documentation for more details.
-        - `trustStore: ARN (arn:aws:elasticloadbalancing:trustStoreArn) | Name (my-trust-store)`
-            - Both ARN and Name of trustStore are supported values.
-            - `trustStore` is required when mode is `verify`.
-        - `ignoreClientCertificateExpiry : true | false (default)`
+    !!!note 
```

Comment:
Can you please update this line as well? https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/main/docs/guide/ingress/annotations.md?plain=1#L61
We want to default value here to N/A since we are not setting from our side.